web/services/hooks/quantiles.tsx

```diff
@@ -12,20 +22,30 @@ const useQuantiles = (data: {
   timeZoneDifference: number;
   metric: string;
 }) => {
+  const filterMap = DASHBOARD_PAGE_TABLE_FILTERS as SingleFilterDef<any>[];
+
+  const userFilters = filterUIToFilterLeafs(filterMap, data.uiFilters);
+
   const {
     data: quantiles,
     isLoading,
     refetch,
   } = useQuery({
-    queryKey: ["quantiles", data.timeFilter, data.metric],
+    queryKey: ["quantiles", data.timeFilter, data.metric, data.uiFilters],
```

Comment:
on line 36, we'll want to grab the queryKey's from the useQuery function, not use the actually variables like `data.timeFilter`. there are some examples of this from the other queries. It looks like this: 

`const quantileTimeFilter = query.queryKey[1] as number`

Reply:
this guarantees that the hook with re-fire if any of those queryKey variables change